src/handlers/detachFromFile.ts

```diff
@@ -0,0 +1,29 @@
+import * as fs from 'fs';;
+import * as path from 'path';
+import * as ejs from 'ejs';
+import {isUndefined,endsWith} from  'lodash';
```

Comment:
please remove isUndefined and endsWith, use !isDefined and string.endsWith instead